src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width,
+        user-scalable=no, initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
+        >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles.css/style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Monoblock"
+      >
+
+      <h1 class="model card__model">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
```suggestion
      <h2 class="model card__model">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```
it's definitely not `h1` - it should be 1 per page
what situation will be in catalog?

Reply:
`"model card__model` - do you need both classes?
Is it a separate block?